lighthouse-core/test/runner-test.js

```diff
@@ -126,6 +128,61 @@ describe('Runner', () => {
       });
   });
 
+  it('rejects when domain name can\'t be resolved', () => {
```

Comment:
It would be great if these could be narrowed in scope by adding them to `gather-runner-test.js` instead and cutting out `Runner` from the tests